### PR TITLE
feat(COMMON,LLD,LLM): Add missing attributes for Polkadot Transaction coming from Wallet API

### PR DESCRIPTION
**Pull request by jdomingos**
### 📝 Description

Adds the missing attributes mapping for the Polkadot Transaction coming from the `wallet-api`, concerning changes made here: https://github.com/LedgerHQ/wallet-api/pull/271

### ❓ Context

We're from [StakeKit](https://www.stakek.it/) and we've just released a Ledger Live app that offers users a unified non-custodial staking experience across a wide range of networks and protocols. We're opening this PR because we would like to offer users the possibility to participate in Polkadot staking via our Ledger Live app.

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.**
- [x] **Impact of the changes:** 
  - This PR impacts how Polkadot transactions are built coming from the Wallet API

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by haammar-ledger — APPROVED** [review]
LGTM :+1: Just needs an approval too from @LedgerHQ/wallet-api team.

File: .changeset/rare-papayas-clean.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+"@ledgerhq/live-common": minor
+"@ledgerhq/live-cli": minor
+"ledger-live-desktop": minor
+"live-mobile": minor
+"web-tools": minor
```
[the remark this answers is not in the document]
Comment (jdomingos) [reply .changeset/rare-papayas-clean.md:6]: Thanks @haammar-ledger! 
They are dependent on the `@ledgerhq/live-common` package and they are build based on the workspace. 
Does this get treated automatically on the releases?

[the remark this answers is not in the document]
Comment (jdomingos) [reply .changeset/rare-papayas-clean.md]: I've updated to have only `common`. Let me know if that is enough :)

**Comment by Justkant** [comment]
Hey I created the branch yesterday just had no time push this until today, @haammar-ledger could take a look at https://github.com/LedgerHQ/ledger-live/pull/5697 instead please ?

**Pull request closed without merging**